pkgs/tools/virtualization/google-compute-guest-agent/0001-oslogin-add-disable-toggle-in-conf-file.patch

```diff
@@ -0,0 +1,40 @@
+From f318f525149b06365e9f0c7f642f09f401c3e1c1 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?F=C3=A9lix=20Baylac-Jacqu=C3=A9?= <felix@alternativebit.fr>
+Date: Fri, 5 Nov 2021 15:40:13 +0100
+Subject: [PATCH] oslogin: add disable toggle in conf file
+
+The guest agent is trying to imperatively alter the sshd authorized
+key config, inject some new NSS configuration, and update the PAM
+config. This won't work on NixOS. We already have a "legacy" support
+for the os-login feature that is more declarative, let's keep on using
+that.
```

Comment:
I think we initially figured out how that's configured, by reading the previous python scripts "installing" this.

We should check if the "instructions" (now living in `google_guest_agent/oslogin.go`) are still the same, or if we need to update something in our declarative version of the module. I assume the nss modules themselves still stayed the same?

Reply:
Ah! I see. Sounds like a plan.

Reply:
Checked this during comparing PRs, our configuration is up to date with one from `oslogin.go`.